

= 30.08.2022 20:42:15 (UTC +8:00) === ECU-TEST 2022.2.125314+1e79c4002501 125314+1e79c4002501 (2022/06/14 12:33:44) (64bit) ===
Python 3.9.7 (tags/v3.9.7:1016ef3, Aug 30 2021, 20:19:38) [MSC v.1929 64 bit (AMD64)]

Roaming License: ECU-TEST
         - License options:
             + Diagnostics add-on
             + Trace Analysis
         - valid up to version: 2022.4
         - valid until: 14-sep-2022
         - licensed for: TURING


2022-08-30 20:42:15.661 9260 ManagedWorker-6*PE.Run() ERROR:
                        Traceback (most recent call last):
                          File "tts\testExecution\engine\TestEngine.py", line 970, in ExecuteTestStep
                          File "tts\testExecution\testSteps\generic\TestStepJobExec.py", line 253, in Run
                        tt._Exceptions_.TTError: An error occurred when calling "ByteStream('00 56 78')": invalid Syntax
2022-08-30 20:42:15.688 9260 ManagedWorker-6*PE.Run() ERROR:
                        Traceback (most recent call last):
                          File "tts\testExecution\engine\TestEngine.py", line 970, in ExecuteTestStep
                          File "tts\testExecution\testSteps\generic\TestStepJobExec.py", line 253, in Run
                        tt._Exceptions_.TTError: An error occurred when calling "ByteStream('00 87 23')": invalid Syntax
2022-08-30 20:42:15.726 9260 ManagedWorker-6*PE.Run() ERROR:
                        Traceback (most recent call last):
                          File "tts\core\report\TestReport.py", line 342, in Process
                          File "tts\testExecution\testSteps\generic\TestStepJobExec.py", line 117, in Report
                        AttributeError: 'NoneType' object has no attribute 'GetValue'
2022-08-30 20:42:15.738 9260 ManagedWorker-6*PE.Run() ERROR:
                        Traceback (most recent call last):
                          File "tts\core\report\TestReport.py", line 342, in Process
                          File "tts\testExecution\testSteps\generic\TestStepJobExec.py", line 117, in Report
                        AttributeError: 'NoneType' object has no attribute 'GetValue'
